bin/testfiles/Bug52310.xml

```diff
@@ -2,16 +2,16 @@
 <testResults version="1.2">
 <sample s="true" lb="ComputeIPAddr" rc="200" rm="OK" dt="text"/>
 <httpSample s="true" lb="HTTP-Request-HC31" rc="200" rm="OK" dt="text">
-  <httpSample s="true" lb="HTTP-Request-HC31-0" rc="200" rm="OK" dt="text"/>
```

Comment:
I don't understand why this test result is modified. 
The result should be numbered sub results no ?

Reply:
From the bug I understand, that JMeter will change subResults labels only if this labels are equals. 
The parent label is `HTTP-Request-HC31`, subResults labels start with `http://jmeter.apache.org/.....` They are not equals and JMeter does not change them.